src/localization/en.json

```diff
@@ -230,7 +230,7 @@
   "questionnaireCard": {
     "questionnairesBuilder": "Questionnaires builder",
     "questionnaires": "Questionnaires",
-    "progressBar": "Progress Bar"   
+    "passedQuestionnaires": "You have passed {0} out of {1} questionnaire"
```

Comment:
As far as I can tell the contents of the curley bracket don't seem to matter, but I suppose it could be more descriptive e.g. {passedQuestionnaires} and {totalQuestionnaires}.
Similar for fi translations. I tested locally and it seemed to work fine but please confirm